boilerplate/fullstack/remix/app/routes/_index.tsx

```diff
@@ -4,71 +4,51 @@ import SuperTokens from "supertokens-auth-react";
 import { LoaderFunctionArgs, redirect } from "@remix-run/node";
 import { useLoaderData } from "@remix-run/react";
 import SessionReact from "supertokens-auth-react/recipe/session/index.js";
-import { SessionDataForUI } from "../lib/superTokensTypes";
 import { getSessionDetails } from "../lib/superTokensHelpers";
 import { TryRefreshComponent } from "../components/tryRefreshClientComponent";
 import { SessionAuthForRemix } from "../components/sessionAuthForRemix";
 
+interface SessionDataForUI {
+    note: string;
+    userId: string;
+    sessionHandle: string;
+    accessTokenPayload: object;
+}
+
 interface SessionForRemixProps {
-    session?: {
-        userId?: string;
-        sessionHandle?: string;
-        accessTokenPayload: SessionDataForUI;
-    };
-    hasInvalidClaims: boolean;
-    hasToken: boolean;
+    userId: string;
+    sessionHandle?: string;
+    accessTokenPayload: SessionDataForUI;
```

Comment:
this is strange:
accessTokenPayload is set to a type which also contains a prop called accessTokenPayload? Also, the userId prop is in the root obj, and in the accessTokenPayload obj?

Reply:
✔️